presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -1023,4 +1085,91 @@ private static String toCharValue(Object value, int minLength)
         }
         return values;
     }
+
+    private class TestingNoInputFilterFunction
+            extends FilterFunction
+            implements TestingFilterFunction
+    {
+        private Function<Integer, Boolean> rowDoer;
```

Comment:
Assume `private Predicate<Integer> predicate;` is a better name + type, where `Predicate` is the java.util one.

Reply:
nice